### PR TITLE
[C-API] remove padded value of dimensions (0-init) @open sesame 08/17 11:02

### PR DESCRIPTION
**Pull request by niley7464**
This patch removes padded value of dimensions.
Dimension in ml_tensor_info_s is 0 by default.

related to : https://github.com/nnstreamer/nnstreamer/pull/4111

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]

File: c/src/ml-api-inference-internal.c [hunk 1]
```diff
@@ -181,18 +184,18 @@ _ml_tensors_info_copy_from_ml (GstTensorsInfo * gst_info,
 
     gst_info->info[i].type = convert_tensor_type_from (ml_info->info[i].type);
 
+    for (j = 0; j < NNS_TENSOR_RANK_LIMIT; j++) {
+      gst_info->info[i].dimension[j] = 0;
+    }
+
```
Comment (myungjoo) [inline c/src/ml-api-inference-internal.c:190]: or.. keep the old code with change of ```s/= 1;/= 0;/```.

**Review by myungjoo — APPROVED** [review]

File: tests/capi/unittest_capi_inference.cc [hunk 2]
```diff
@@ -4754,7 +4744,7 @@ TEST (nnstreamer_capi_element, get_property_string_04_n)
 
   pipeline = g_strdup_printf (
       "appsrc name=appsrc ! "
-      "other/tensor,dimension=(string)1:1:1:1,type=(string)float32,framerate=(fraction)0/1 ! "
+      "other/tensor,dimension=(string)1,type=(string)float32,framerate=(fraction)0/1 ! "
       "tensor_filter name=filter_h framework=tensorflow-lite model=%s ! tensor_sink name=tensor_sink",
```
Comment (myungjoo) [inline tests/capi/unittest_capi_inference.cc:4748]: For such testing, let's keep the previous test cases and add new cases with shortened dimension declarations.

Comment (myungjoo) [reply tests/capi/unittest_capi_inference.cc:4748]: Let's talk about it tomorrow anytime when I'm available in the office.

**Review by jaeyun-jung — APPROVED** [review]

**Comment by jaeyun-jung** [comment]
plz fix indent and rebase to merge this PR.

**Review by songgot — APPROVED** [review]
LGTM :+1:

**Comment by jaeyun-jung** [comment]
need fix build error after update your PR :)

**Comment by niley7464** [comment]
> need fix build error after update your PR :)

Sorry 😢
Fixed it

**Pull request merged**